docs/docs/kds-rest-api.md

```diff
@@ -49,13 +50,15 @@ a cryptographic link between the client and the enclave.
 }
 ```
 
-| Field | Description |
-| ----- | ----------- |
-| publicKey | The Base64 encoded public key. The caller should verify this public key using the signature to ensure it was returned by the KDS enclave. |
-| signature | A signature in Base64 that can be used with the `kdsAttestationReport` to verify that the public key was returned by the KDS enclave. This verification is crucial for security reasons. For more details refer to [`Appendix A`](#appendix-a-signature-verification).|
-| kdsAttestationReport | The `EnclaveInstanceInfo` of the KDS enclave in Base64. The application enclave should validate this report before checking the public key signature to ensure the public key was returned by the KDS enclave. |
-
+| Field | Description                                                                                                                                                                                                             |
+| ----- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| publicKey | A [Base64](https://en.wikipedia.org/wiki/Base64) encoded public key.                                                                                                                                                    |
+| signature | A signature in Base64 that the caller can use with the `kdsAttestationReport` to verify that the KDS enclave returned the public key. This verification is [crucial for security reasons](#public-key-integrity-check). |
+| kdsAttestationReport | The `EnclaveInstanceInfo` of the KDS enclave in Base64. The application enclave needs to validate this report before checking the public key signature to ensure that the the KDS enclave returned the public key. |
 
+The client can access this Curve25519 public key irrespective of the enclave constraints. In contrast, the
+KDS provides a Curve25519 private key material to the application enclave only if it meets the constraints defined 
+by the client.
```

Comment:
I think this can be removed, as it's covered in the description of the constraint field in the request body (as per my previous comment).